portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -19,14 +19,39 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.*;  
+
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("Hello John!");
+    
+    // make arraylist with three hard-coded values
+   ArrayList<String> comments = new ArrayList<String>(); // Creating arraylist 
```

Comment:
```suggestion
   List<String> comments = new ArrayList<String>(); // Creating arraylist 
```

Better to use interface for holding the result.

Reply:
I received this error when switching ArrayList<String> to List<String>: 

[ERROR] /home/jjara/software-product-sprint/portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:[
45,34] cannot find symbol
  symbol:   class List

Reply:
You need to import it explicitly now that you have changed the import to just for Arraylist